### PR TITLE
display products gallery

### PR DESCRIPTION
**Pull request by gitygity**

### CODE REVIEW

File: gity_startingup2017_bot/gity_shopbot.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+def start_method(bot,update):
+    bot.sendMessage(update.message.chat_id,"hellow user!!" )
+
+
+def sendImage(bot,update):
+    chat_id = update.message.chat_id
+    url = "http://www.samatak.com/image/2016/09/2/1033807340-samatak-com.jpg";
+    img =io.BytesIO(urllib.request.urlopen(url).read())
+    bot.send_photo(chat_id, img, 'محصول شماره ۱: قیمت ۲۵۰۰۰۰')
```
Comment (ahangarha) [inline gity_startingup2017_bot/gity_shopbot.py]: You can just pass the URL to `send_photo`. You don't need download it. Passing just URL, would speed-up the process and also save your resources.

File: gity_startingup2017_bot/gity_shopbot.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+from telegram.ext import Updater, CommandHandler
+import urllib.request,io
+
+
+updater=Updater('TOKEN')
+
+def main_menu(bot,update):
+    menu=[['تماس با ما','درباره ما','گروه محصولات']]
+
+    bot.sendMessage(update.message.chat_id,"از منوی زیر بخش مورد نظر را انتخاب کنید:",reply_markup=ReplyKeyboardMarkup(menu))
```
Comment (ahangarha) [inline gity_startingup2017_bot/gity_shopbot.py]: See [This topic](http://ask.startingup.ir/t/topic/18)

File: gity_startingup2017_bot/gity_shopbot.py (hunk 1 above)
Comment (gitygity) [reply gity_startingup2017_bot/gity_shopbot.py]: You mean i should set url of image Instead of img in parenthesis?

Comment (ahangarha) [reply gity_startingup2017_bot/gity_shopbot.py]: Exactly. If you want to use a url to send photo, simply send its url. You don't need to get the file downloaded on your system and upload again each time. this means you waste your bandwidth twice the file size.

File: gity_startingup2017_bot/gity_shopbot.py (hunk 2 above)
Comment (gitygity) [reply gity_startingup2017_bot/gity_shopbot.py]: ok thx . i chaged them

File: gity_startingup2017_bot/gity_shopbot.py (hunk 1 above)
Comment (gitygity) [reply gity_startingup2017_bot/gity_shopbot.py]: i will do this .just  i hope have not any error

**Review by masihtehrani — COMMENTED** [review]
please change readme , and add your bot in readme

**Pull request merged**